examples/spread-in-binding-patterns/spread_in_binding_patterns.bal

```diff
@@ -0,0 +1,18 @@
+import ballerina/io;
+
+type Id [int, string...];
+
+function process(Id id) {
+    // The `id` is destructured with a binding pattern such that the `path` contains all the rest members
+    // of the `id` tuple.
+    var [n, ...path] = id;
```

Comment:
Is `path` (or `paths`) the best name here? I'd rather use something like `names`. Can we improve the variable names in general?